fudge-ratio/java/servlets/FudgeRatio.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+@WebServlet("/data")
+public class DataServlet extends HttpServlet {
+
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    }
+
+
+    /**
+     * Changes the fudege ratio (actual/expected time) by converting start and end time to seconds.
+     * @param start   This is the user's start time for a task. 
+     *                This is a string in the format "00:00:00" (hrs:min:second)
+     *
+     * @param end     This is the end time of the task, in the 
+     *                same format as the start string.
+     */
+    private double calculateRatio(String start, String end, int expected){
```

Comment:
We could also pass the arguments as Date instead of String to make it easier and robust (but this is just an idea, String works just fine the way you did it)

Reply:
When I wrote thus function, Jerry said that he was planning on sending time as a string, but that might change.